lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * Version: `ES6`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireShorthandArrowFunctions": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * evens.map(v => v + 1);
+ * evens.map(v => {
+ *     v = v + 1;
```

Comment:
Do you mean this specific example or the rule? I can remove this example if it's confusing.